pkg/frontend/transport/handler.go

```diff
@@ -348,6 +348,12 @@ func (f *Handler) reportQueryStats(r *http.Request, userID string, queryString u
 	if queryPriority := r.Header.Get(util.QueryPriorityHeaderKey); len(queryPriority) > 0 {
 		logMessage = append(logMessage, "priority", queryPriority)
 	}
+	if dataFetchMinTime := r.Header.Get(util.DataFetchMinTime); len(dataFetchMinTime) > 0 {
+		logMessage = append(logMessage, "data_fetch_min_time", dataFetchMinTime)
+	}
+	if dataFetchMaxTime := r.Header.Get(util.DataFetchMaxTime); len(dataFetchMaxTime) > 0 {
+		logMessage = append(logMessage, "data_fetch_max_time", dataFetchMaxTime)
+	}
```

Comment:
We might be able to use querier stats to propagate this through context. But we don't have to change it in this pr. We can probably do it later

Reply:
I'll try to use context of the request.

One more question, what do u think of replacing `data_fetch_min_time` with `mint` and `data_fetch_max_time` to `maxt`, simliar to the tsdb logs?

Reply:
Let's use querier stats instead since it is a more organized struct.
Do we think it is clear for users to differentiate `mint` and `maxt` from `start` and `end`? Which one is for the range query param and which for actual data fetched time range?
I think it is fine for TSDB logs to use `mint` and `maxt` because it is not aware of the range queries params. But query frontend is aware of both so let's better be specific.

Reply:
Changed them to `data_select_max_time` and `data_select_min_time`